README.md

```diff
@@ -22,12 +33,134 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
+
+```
+pip install smdebug
+```
+
+### Supported Frameworks
+There are two ways in which you can enable SageMaker Debugger while training on SageMaker&mdash;Zero Script Change and Bring Your Own Training Container.
+
+#### Zero Script Change
+
+You can use your own training script while using [AWS Deep Learning Containers (DLC)](https://aws.amazon.com/machine-learning/containers/) in TensorFlow, PyTorch, MXNet, and XGBoost frameworks. The AWS DLCs enable you to use Debugger with no changes to your training script by automatically adding SageMaker Debugger's `Hook`.
+The following table shows currently supported versions of the four frameworks for Zero Script Change experience.
+
+| Framework | Version |
+| --- | --- |
+| [TensorFlow](docs/tensorflow.md) | 1.15, 1.15.2, 2.1 |
+| [MXNet](docs/mxnet.md) | 1.6 |
+| [PyTorch](docs/pytorch.md) | 1.3, 1.4 |
+| [XGBoost](docs/xgboost.md) | >=0.90-2 [As Built-in algorithm](xgboost.md#use-xgboost-as-a-built-in-algorithm)|
+
+For the full list and information of the AWS DLCs, see [Deep Learning Containers Images](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-images.html#deep-learning-containers-images-table).
+
+
+#### Bring Your Own Training Container
+
+`smdebug` supports frameworks other than the ones listed in the previous Zero Script Change section. You can use your own training script by adding a minimal modification.
+Currently supported versions of frameworks are listed in the following table.
+
+| Framework | Versions |
+| --- | --- |
+| [TensorFlow](docs/tensorflow.md) | 1.14. 1.15, 2.0.1, 2.1.0 |
+| Keras (with TensorFlow backend) | 2.3 |
+| [MXNet](docs/mxnet.md) | 1.4, 1.5, 1.6 |
+| [PyTorch](docs/pytorch.md) | 1.2, 1.3, 1.4 |
+| [XGBoost](docs/xgboost.md) | [As Framework](xgboost.md#use-xgboost-as-a-framework) |
+
+### Support for Distributed Training and Known Limitations
+
+<table>
+    <thead>
+        <tr>
+           <th colspan=3>
+           Distributed Training
+           </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=2>Horovod</td>
+            <td>Supported</td>
+            <td>TF 1.15, PT 1.4, MX 1.6</td>
+        </tr>
+        <tr>
+            <td>Not supported</td>
+            <td>TF 2.x, PT 1.5</td>
+        </tr>
+        <tr>
+            <td>Parameter Server-based</td>
+            <td colspan=2>Not supported</td>
+        </tr>
+    </tbody>
+</table>
+
+## How It Works
+
+Amazon SageMaker Debugger uses the construct of a `Hook` to save the values of requested tensors throughout the training process. You can then setup a `Rule` job which simultaneously monitors and validates these tensors to ensure
+that training is progressing as expected.
+
+A `Rule` checks for vanishing gradients, exploding tensor values, or poor weight initialization. `Rule`s are attached to CloudWatch events, so that when a rule is triggered it changes the state of the CloudWatch event.
```

Comment:
```suggestion
A `Rule` checks for vanishing gradients, exploding tensor values, or poor weight initialization. `Rule`s are attached to Amazon CloudWatch events, so that when a rule is triggered it changes the state of the CloudWatch event.
```
1st use of CloudWatch